### PR TITLE
chore: project addition by mohitparmar1 #1332

### PR DESCRIPTION
**Pull request by mohitparmar1**
close issue #1332

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -2684,5 +2684,21 @@
...
+    "YouTube": "#",
+    "Twitter": "https://twitter.com/mohitparmar1_"
+  },
+  "Projects": [
+    {
+      "link": "https://github.com/mohitparmar1/Followers-Animation",
+      "title": "Followers Animation",
+      "description": "its a followers increasing animation ",
+      "tech": ["Html","Css","javascript"]
+    }
```
Comment (priyankarpal) [inline src/DB/projects.json:2701]: you have already added projects before. So add the code to the previous one. Check contributing. md for more information.

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by priyankarpal** [comment]
also. there are some conflicts errors please resolve

File: src/DB/projects.json (hunk 1 above)
Comment (mohitparmar1) [reply src/DB/projects.json]: done changes

**Review by priyankarpal — CHANGES_REQUESTED** [review]
I noticed that your repository doesn't contain any code; it only consists of a README file. As per the guidelines outlined in GSSoS'23, this is considered invalid. Therefore, if you persist in posting such spam, this pull request will be closed, and there's a possibility that you may be blocked.

image

**Comment by priyankarpal** [comment]
also, there are  no contribution guidelines, license, code of conduct  

image

File: src/DB/projects.json [hunk 2]
```diff
@@ -2623,7 +2623,16 @@
         "title": "Leads Tracker",
         "description": "its a tracker that track our important tabs",
         "tech": ["html", "css", "javascript"]
-      },
+      }
```
Comment (priyankarpal) [inline src/DB/projects.json:2626]: add a `,`  
```suggestion
      },
```

File: src/DB/projects.json [hunk 3]
```diff
@@ -2623,7 +2623,16 @@
...
+      "Projects": [
```
Comment (priyankarpal) [inline src/DB/projects.json:2627]: remove 
```suggestion
 
```

File: src/DB/projects.json [hunk 4]
```diff
@@ -2623,7 +2623,16 @@
...
+    {
+      "link": "https://github.com/mohitparmar1/Followers-Animation",
+      "title": "Followers Animation",
+      "description": "its a followers increasing animation ",
+      "tech": ["Html","Css","javascript"]
+    }
+    ]
```
Comment (priyankarpal) [inline src/DB/projects.json:2634]: remove 
```suggestion
    
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by mohitparmar1** [comment]
closing this pr and making new changes

**Pull request closed without merging**